concent_api/core/admin.py

```diff
@@ -56,9 +67,14 @@ def get_requestor_public_key(self, obj):  # pylint: disable=no-self-use
         return obj.requestor.public_key
     get_requestor_public_key.short_description = 'Requestor public key'  # type: ignore
 
+    def get_queryset(self, request):
+        return Subtask.objects_with_timing_columns
 
-class PendingResponseAdmin(ModelAdminReadOnlyMixin, admin.ModelAdmin):
+    def download_deadline(self, obj):  # pylint: disable=no-self-use
```

Comment:
This should have `short_description` like the methods above.

Reply:
Done.